src/libs/LoginUtils.ts

```diff
@@ -92,12 +57,12 @@ function validateNumber(values: string): string {
  * Check number is valid and attach country code
  * @returns a valid phone number with country code
  */
-function getPhoneLogin(partnerUserID: string): string {
+function getPhoneLogin(partnerUserID: string, countryCode: number): string {
```

Comment:
Should we also update the test for this method? As well as the other method which got the new parameter?